scene/resources/importer_mesh.cpp

```diff
@@ -1292,7 +1293,15 @@ Error ImporterMesh::lightmap_unwrap_cached(const Transform3D &p_base_transform,
 		Ref<SurfaceTool> &tool = surfaces_tools[i];
 		tool->index();
 		Array arrays = tool->commit_to_arrays();
-		add_surface(tool->get_primitive_type(), arrays, Array(), Dictionary(), tool->get_material(), tool->get_meta("name"), lightmap_surfaces[i].format);
+
+		uint64_t format = lightmap_surfaces[i].format;
```

Comment:
I made this `uint64_t` instead of `uint32_t` as future proofing for https://github.com/godotengine/godot/pull/81138